### ISSUE TITLE
local voice not showing in the list after installing

### ISSUE THREAD

**Issue by t-node** [ken107/read-aloud#264]
I bought voice from harpo software because the online voices from amazon is very laggy and takes time. After installing the voice it doesn't show in the extension.I used to work in older version of chrome before version 90..

**Comment by ken107** [comment]
SAPI 5 voices stopped showing up in browsers some time ago due to some Windows update.  I don't know of the cause, or any workaround.  Sorry

**Comment by ken107** [comment]
I see, I really don't know,  I just now tried installing Brian voice on my WIndows 10 and can't get it to appear in Firefox or any browser.  This is something at the browser and operating system level, and not something that can be done within the extension.  The browser isn't recognizing those voices, and they're not making them available to extensions.

**Comment by t-node** [comment]
sure thanks a lot for your help, Problem with Amazon/Google voices is that they are very laggy due to network round trip time I guess. Voices installed on the system are very fast. The voice I am using is bought from Harpo only.

My Firefox is version 99 (64 bit) and OS is Windows 10 Enterprise

**Comment by ken107** [comment]
If you install earlier version of Chrome now and see the voices, then it's definitely a problem with Chrome and not with Windows.  Yea, it's either one of those two things.

**Comment by t-node** [comment]
![image](https://user-images.githubusercontent.com/1595069/162677121-2ea22821-f7e7-4522-b0b2-a0d1b12cb5ea.png)


This is Joey voice on Firefox for your reference

**Comment by ken107** [comment]
Yes.  However, there's nothing we can do.  Chrome doesn't detect that voice.  You may want to file a bug with the Chromium project instead.

**Comment by ken107** [comment]
Current situation:
SAPI5 voices returned fully to Firefox
Chrome only shows Microsoft SAPI5 voices (i.e. David, Zira)
Edge shows none (but Edge provides a huge collection of high quality Microsoft Online Natural voices in their place)

**Comment by luc42ei** [comment]
My fork filters the voice list by configured credentials: https://github.com/luc42ei/read-aloud

Amazon Polly voices are hidden unless AWS keys are entered, Google Wavenet hidden without a GCP key, etc. So you only see voices that will actually work with your current setup.
